lib/Command/V1.pm

```diff
@@ -831,6 +831,11 @@ sub help_options {
             $default_value = "\nDefault value $default_value if not specified";
         }
 
+        my $calculated_default = $property_meta->calculated_default;
+        if (defined $calculated_default) {
```

Comment:
Good though.  Yeah, the [initializer](https://github.com/genome/UR/blob/aa0b83125df7d8eec1ced630003ff4c21ac610e0/lib/UR/Object/Type/Initializer.pm#L1095-L1097) does block that.